magneto:
  rl:
    active:                               True
  simulation:
    # - Sets params related to sim behavior (from original work)
    walkset:                              config/Magneto/USERCONTROLWALK.yaml
    # - Magnetism map making
    magnetism_map:
      # - In pixel corrdinates, relative to standard pixel indexing
      relative_starting_postion:
        x:                                False # TODO
        y:                                False # TODO
      # - Size of wall we are climbing
      wall_geometry:                             
        height:                           5 # m
        width:                            5 # m
      # - Length of each pixel from image representation in m
      image_resolution:                   0.15 # m
    # - Duration in seconds physics are unpaused when an action is triggered
    resume_duration:                      2 # seconds
    # - For auto clicking to maximize and close sim window
    vertical_pixel_calibration_offset:    0 #24 # 0 for my monitors, 24 for my laptop
    # - From Magneto description, used to assign actions to a specific link
    link_idx:
      AR:                                 23
      AL:                                 14
      BL:                                 32
      BR:                                 41
